charts/backstage/values.yaml

```diff
@@ -170,10 +208,18 @@ upstream:
         adminPasswordKey: postgres-password
         userPasswordKey: password
     primary:
+      # TODO: https://issues.redhat.com/browse/RHIDP-2645
```

Comment:
Jira https://issues.redhat.com/browse/RHIDP-2645 details implementing the podSecurityContext and containerSecurityContext settings once the dependencies for the backstage dependency are updated to support these settings.